stile/hsc/base_tasks.py

```diff
@@ -1358,3 +1378,11 @@ def _makeArgumentParser(cls):
         parser.description = parser_description
         return parser
 
+    def getCalibData(self, dataRef, shape_cols):
```

Comment:
Same question here as above: why duplicate the code rather than letting inheritance handle it?